.pre-commit-config.yaml

```
@@ -0,0 +1,36 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: fix-encoding-pragma
+        exclude: tests/test_data
+      - id: trailing-whitespace
+        exclude: tests/test_data
+      - id: end-of-file-fixer
+        exclude: tests/test_data
+      - id: check-docstring-first
+      - id: debug-statements
+      - id: check-toml
+      - id: check-yaml
+        exclude: tests/test_data
+      - id: requirements-txt-fixer
+      - id: detect-private-key
+      - id: check-merge-conflict
+
+  - repo: https://github.com/psf/black
+    rev: 24.4.2
+    hooks:
+      - id: black
+        exclude: tests/test_data
+      - id: black-jupyter
```

```suggestion
      - id: end-of-file-fixer
      - id: check-docstring-first
      - id: debug-statements
      - id: check-toml
      - id: check-yaml
      - id: requirements-txt-fixer
      - id: detect-private-key
      - id: check-merge-conflict

  - repo: https://github.com/psf/black
    rev: 24.4.2
    hooks:
      - id: black
      - id: black-jupyter
```

The excludes are not needed here.